builder/dockerfile/dispatchers_test.go

```diff
@@ -370,20 +370,6 @@ func TestStopSignal(t *testing.T) {
 	assert.Check(t, is.Equal(signal, sb.state.runConfig.StopSignal))
 }
 
-func TestArg(t *testing.T) {
```

Comment:
Why was the test removed instead of being updated?

Reply:
Ah, just want to have a quick CI around the available arches 😄